viewPost.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+include('config/init.php');
+
+
+$result=getAllBlogPosts();
+$blogPosts= getBlogPost($_GET['categoryId']);
+
+
+echoHeader("".$blogPosts['title'], "".$blogPosts ['description']);
+
+echo "
+<h2>".$blogPosts['title']."</h2>
+<p>".$blogPosts['content']."</p> 
+";
+
+if(isset($_REQUEST['newComment'])){
```

Comment:
For similar reasons to why you always put the `include` at the top of the file, you should always process forms before echoing any html. And as mentioned above, I'd suggest redirecting the page after saving a comment so they can't double-submit be reloading the page.